doc/conf.py

```diff
@@ -0,0 +1,44 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'python-dbusmock'
+copyright = '2023, Martin Pitt'  # noqa: A001
+author = 'Martin Pitt'
+release = '0.29.1'
```

Comment:
This needs to use the dynamic version like [here](https://github.com/martinpitt/python-dbusmock/blob/main/dbusmock/__init__.py#L20).

Reply:
Hmm, does this even appear anywhere? I don't see it on https://whot.github.io/python-dbusmock/index.html or https://whot.github.io/python-dbusmock/autoapi/dbusmock/index.html or https://whot.github.io/python-dbusmock/search.html?q=0.29.1

Reply:
I had to grep the source code and yes, it does show up:
`  <title>python-dbusmock &mdash; python-dbusmock 0.29.1 documentation</title>`

Guess with everyone having a thousand tabs open no-one would notice that ;)

Reply:
Locally the dynamic import works in the sense of sphinx-build succeeding. I pushed that.

I also added the two new directories to .gitignore.